nautobot_netbox_importer/tests/test_import_objectchange.py

```diff
@@ -0,0 +1,121 @@
+"""Test the importing functionality of nautobot-netbox-importer."""
+
+import os
+import json
+import copy
+from unittest.mock import Mock
+
+import yaml
+from django.core.management import call_command
+from django.contrib.contenttypes.models import ContentType
+from django.test import TestCase
+from nautobot.extras.models import ObjectChange
+from nautobot.extras import models
+from nautobot_netbox_importer.management.commands.import_netbox_objectchange_json import Command
+from .test_import import TestImport
+
+
+NETBOX_DATA_FILE = os.path.join(os.path.dirname(__file__), "fixtures", "netbox_dump.json")
+NETBOX_OBJECTCHANGE_DATA_FILE = os.path.join(os.path.dirname(__file__), "fixtures", "netbox_objectchange_dump.json")
+NAUTOBOT_DATA_FILE = os.path.join(os.path.dirname(__file__), "fixtures", "nautobot_expectations.yaml")
+NAUTOBOT_OBJECTCHANGE_DATA_FILE = os.path.join(
+    os.path.dirname(__file__), "fixtures", "nautobot_objectchange_expectations.yaml"
+)
+
+
+class TestImportObjectChange(TestImport):
+    """Test the importing functionality of nautobot-netbox-importer, for ObjectChange objects.
+
+    It is mostly reusing same tests as TestImport, but with an extra command during TestData Setup.
+    """
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        """One-time setup function called before running the test functions in this class."""
+        # First we import the data without ObjectChanges
+        call_command("import_netbox_json", NETBOX_DATA_FILE, "2.10.4", verbosity=0)
```

Comment:
I recently encountered some issues where `call_command` seems to be bypassing the test database and interacting directly with the development database. Please take a look at the changes I made to the existing tests in https://github.com/nautobot/nautobot-plugin-netbox-importer/pull/43/commits/e35e9f8a47f0105cc707464e952aff933ee3cc3c#diff-7d7ec2b55861a28ecdd4f6d9df3c7d4cdc38c6af12bdde27ef84f42406d5b220 to address this and see if they can be used here as well.